bhardwajdisha/Rocket.Chat
# FEAT: quick action button to initiate the verification process

## Proposed changes (including videos or screenshots)
A quick action button accessible to only livechat manager, agents and admins, to allow them to start the verification process to check if the visitor is a bot or not.

https://github.com/bhardwajdisha/Rocket.Chat/assets/67470541/5c68357d-fbe7-4b92-86e4-2773327dbdb6







## Issue(s)


## Steps to test or reproduce

1. Go to Omnichannels
2. Join any ongoing chat
3. From the quick action button select verify-user button

## Further comments

A rocket cat will send hi to the visitor - A sample message used for testing
